### DISCUSSION TITLE
How do i trigger suggestions on copilot

### DISCUSSION THREAD

**Discussion by chuma1409** [community/community#68052] [Copilot Conversations]
### Select Topic Area

Question

### Body

how do i trigger suggestions on copilot i use vscode on ubuntu

**Comment by worldwidewurf** [comment]
Accept inline code suggestion — Tab
Dismiss inline code suggestion — Esc
Show next suggestion — Alt + ] or Option (⌥) + ]
Show previous suggestion — Alt + [ or Option (⌥) + [
Trigger suggestion — Alt + \ or Option (⌥) + \
Open ten suggestions in a separate pane — Ctrl + Enter

**Reply by abdullzz** [reply]
more often, it will not show any inline code suggestion

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
